wordpress-mobile/WordPress-Android
# Jetpack post preview

Fixes #5326. This feature turned out to much simpler than what I understood it to be. Don't let the branch name fool you, we can't show authenticated version of a post without password for Jetpack sites. That was my misunderstanding.

This PR fixes the Jetpack blog posts not showing up at all in webview. We used to show a username/password error, because of treating a Jetpack site the same as self-hosted. We can only show the regular url of the post, and to handle previews we add a `frame-nonce` parameter to the url.

Relevant FluxC PR: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/358

To test:
* Create a draft in a Jetpack site
* Tap on the "View" button in posts list, you should see the post even if it's a draft (remote one, of course)
